package.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
- [ ] It would be nice if you add your package-lock.json file in the .gitignore file because it was giving me an error on my browser about a package conflict so you can prevent that if everyone who would clone your repository would have one's own package-lock.json file :+1: Secondly this file is not mandatory to be in your repository because when package.json file is keeping track of your packages then package-lock.json file is unnecessary here :thinking: